brave/src/main/java/com/linecorp/armeria/client/brave/BraveClient.java

```diff
@@ -175,10 +175,15 @@ private void maybeAddTagsToSpan(ClientRequestContext ctx, HttpClientRequest brav
                               timings.socketConnectDurationNanos());
                 }
                 if (timings.pendingAcquisitionDurationNanos() != -1) {
-                    logTiming(span, "connection-reuse.start", "connection-reuse.end",
+                    logTiming(span, "connection-pending.start", "connection-pending.end",
```

Comment:
This is a breaking change, but I think it's relatively low impact.
Let me know if anyone feels otherwise